gulpfile.js

```diff
@@ -92,7 +107,7 @@ var frontendDependencies = manifest.dependencies.frontend;
 var cssFilesPath = [];
 var jsFilesPath = [];
 var fontFolderPath = [];
-var cssBackgroundPath = [];
+var cssBackgroundFilepaths = [];
 var generatedTargetDir = path.join(
   'third_party', 'generated',
   isMinificationNeeded ? 'prod' : 'dev');
```

Comment:
FYI, my question about lines 117 to 145 still stands -- should we wrap this in a function and call it from the tasks that need it, instead of running it all the time? Seems more modular to me, especially as this file gets bigger.